auth0/v3/management/connections.py

```diff
@@ -128,5 +128,4 @@ def delete_user_by_email(self, id, email):
         Returns:
             An empty dict.
         """
-        return self.client.delete(self._url(id) + '/users', params={'email': email})
-
+        return self.client.delete('%s/users' % self._url(id), params={'email': email})
```

Comment:
@vkmrishad - This jumped out at us ... this is not a very clear way to represent that URL. I would change this to do what we're doing here:

https://github.com/auth0/auth0-python/blob/master/auth0/v3/management/users.py#L169

Reply:
@joshcanhelp 
For your information %s formatting is a deprecated method. Actually, I thought to use placeholder {} formatting or f-string formation.
I used %s because is mostly used in this project.

Reply:
If its ok, we can change all formatting from %s to "{}.format()"